core/drivers/crypto/crypto_api/include/drvcrypt_acipher.h

```diff
@@ -133,6 +133,17 @@ struct drvcrypt_secret_data {
 	struct drvcrypt_buf secret; /* Shared secret */
 };
 
+/*
+ * Cipher data
+ */
+struct drvcrypt_cipher_data {
+	uint32_t algo;                  /* Operation algorithm */
+	void *key;                      /* Public or Private Key */
+	size_t size_sec;                /* Security size in bytes */
+	struct drvcrypt_buf plaintext;  /* Message to encrypt */
```

Comment:
The struct can be used for decryption, so prefer `/* Clear text message */` here.